main.pm

```diff
@@ -423,6 +423,7 @@ sub load_consoletests() {
             loadtest "console/zypper_ar.pm";
         }
         loadtest "console/zypper_ref.pm";
+        loadtest "console/a2ps.pm"; # a2ps is available by default only in openSUSE*
```

Comment:
please move that comment into the test itself, the main.pm is openSUSE specific anyway